server.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const express = require('express');
+const exphbs = require('express-handlebars');
+const methodOverride = require('method-override');
+const bodyParser = require('body-parser');
+const validator = require('./middleware/validate');
+const productDB = require('./routes/products');
```

Comment:
ProductDB and articleDB are not good names where these. Since these are routes and not DB objects.